core.py

```diff
@@ -11,7 +11,10 @@ def list_download_models(cache_dir):
 
 def remove_dir(path):
     try:
-        shutil.rmtree(os.path.join(path, "/*"))
-        print(f"Directory '{path}' successfully removed.")
+        for folder in os.listdir(path):
+            if folder != "models--openai-community--gpt2":
```

Comment:
Do not use static value. Define default model once in webui.py & use it everywhere